Doc/library/shutil.rst

```diff
@@ -74,6 +80,9 @@ Directory and files operations
       Raise :exc:`SameFileError` instead of :exc:`Error`.  Since the former is
       a subclass of the latter, this change is backward compatible.
 
+   .. versionchanged:: 3.7
+     Added `length` parameter
```

Comment:
Indentation should use three spaces here:

```rst
.. versionchanged:: 3.7
   Added the *length* parameter.
```

Reply:
Done.